plugins/org.locationtech.udig.feature.editor/src/org/locationtech/udig/feature/panel/FeaturePanelRegistry.java

```diff
@@ -55,11 +53,11 @@
 @SuppressWarnings("unused")
 public class FeaturePanelRegistry {
 
-    private static final String NO_TAB_ERROR = "Schema {0} declares non-existing tab {1}";
+    private static final String NO_TAB_ERROR = "Schema {0} declares non-existing tab {1}"; //$NON-NLS-1$
 
-    private static final String CONTRIBUTOR_ERROR = "Contributor {0} cannot be created.";
+    private static final String CONTRIBUTOR_ERROR = "Contributor {0} cannot be created."; //$NON-NLS-1$
 
-    private static final String TAB_ERROR = "Tab in {0} declares non-existing category {1}.";
+    private static final String TAB_ERROR = "Tab in {0} declares non-existing category {1}."; //$NON-NLS-1$
```

Comment:
Looks like these constants are used to display messages in error view as IStatus (or at least in an error dialog). Therefore these should be translatable and might be exported to allow translations using different properties files